core/domain/app_feedback_report_constants.py

```diff
@@ -65,7 +65,7 @@ class ReportType(enum.Enum):
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
 # throughout the codebase according to the coding style guide.
 # https://github.com/oppia/oppia/wiki/Coding-style-guide
-class CATEGORY(enum.Enum): # pylint: disable=invalid-name
+class Category(enum.Enum): # pylint: disable=invalid-name
```

Comment:
Here and below, should you be dropping the comments above?

Also, I don't think you've changed the values according to the comment. Maybe worth taking a look at the other enum.Enum classes in the codebase.

Reply:
All the pylint comments next to the class and variables should be removed?

Reply:
No, I'm talking about the ones similar to the comment _above_ this line that this PR is (presumably) solving?